data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -2020,6 +2020,333 @@
+        ],
+        "prob": 2
+      },
+      {
+        "collection": [
+          {
+            "item": "glock_22",
+            "ammo-item": "40sw",
+            "prob": 100,
+            "charges": [ 0, 15 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_glock_22" }
+        ],
+        "prob": 30
+      },
+      {
+        "collection": [
+          {
+            "item": "px4_40",
+            "prob": 100,
+            "charges": [ 0, 14 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_px4_40" }
+        ],
+        "prob": 5
+      },
+      {
+        "collection": [
+          {
+            "item": "sig_40",
+            "prob": 100,
+            "charges": [ 0, 12 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_sig_40" }
+        ],
+        "prob": 10
+      },
+      {
+        "collection": [
+          {
+            "item": "glock_21",
+            "prob": 100,
+            "charges": [ 0, 13 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_glock_21" }
+        ],
+        "prob": 10
+      },
+      {
+        "collection": [
+          {
+            "item": "usp_45",
+            "variant": "usp_45",
+            "prob": 100,
+            "charges": [ 0, 12 ],
+            "damage": [ 0, 3 ],
+            "contents-group": "police_pistol_mods",
+            "dirt": [ 0, 7050 ]
+          },
+          { "group": "field_usp_45" }
+        ],
+        "prob": 2
+      }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_pistols_damaged_extramags_holstered",
+    "//": "Holstered version, necessary for spawning correctly on zombie cop belts, the magazines 'spill' out into the mag pouches on the belt.",
+    "container-item": "holster",
+    "on_overflow": "spill",
+    "subtype": "collection",
+    "items": [ { "group": "police_pistols_damaged_extramags", "prob": 100 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_longarms",
+    "//": "Rifles/Shotguns that US police would have",
+    "subtype": "distribution",
+    "items": [ { "group": "police_carbines", "prob": 80 }, { "group": "police_shotguns", "prob": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "police_weapons",
+    "//": "A police officer's primary and secondary weapon",
+    "subtype": "collection",
+    "items": [ { "group": "police_longarms", "prob": 100 }, { "group": "police_pistols", "prob": 100 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "SWAT_carbines",
+    "//": "Carbines that a US SWAT team would use",
+    "subtype": "distribution",
+    "entries": [
```

Comment:
I'd like to leave the SCAR 17 to a separate group "SWAT_sniper" as that seems to be the role it is used in.
As for the others, I'd like to see some evidence of usage in the Eastern US before I put them in. 
Kevin insists on only having SWAT weapons found on teams in the Eastern US, so I need to make sure anything going in has evidence.

Reply:
The Scar16, I mean.

Reply:
https://imgur.com/UlCCAHw

State police of Maine

Reply:
>Kevin insists on only having SWAT weapons found on teams in the Eastern US, so I need to make sure anything going in has evidence that I can point to.

I believe you're also interpreting Kevin incorrectly, in that the LAPD is such an outlier in terms of armament as opposed to other police departments that it should not be considered. Researching what police have, let alone what their SWAT teams have, is difficult. If evidence can be produced as to what they have, that's well and good, but otherwise we should make a reasonable guess.

I feel I have provided reasonable guesses for armaments.

Reply:
I'd really like to point out that if you check LAPD inventory and compare it to images of SWAT in numerous other states that the majority of the **WEAPONS** they use are not actually as much of an outlier as initial research may suggest.  (AR-15, Remington 700, M4, etc)

(This was certainly not the case even ~10 years ago, but has increasingly become the case with the increasing militarization of even smaller police forces in the last decade or so).

Regardless, Kevin has already indicated "I'm not merging item distributions based on LAPD numbers, full stop". 
Consequently, the SWAT groups are not my focus in this PR anymore.  I'm also not planning on changing their spawns at the moment.  I'll see if I can enable flashlights to drop into whatever itemgroups they are currently using.

Reply:
I also want to clarify I am not looking to totally cross them off the table.  The PR is large enough as it is though, and if I come back to them it will be after this is approved.